scripts/protocgen.sh

```diff
@@ -8,16 +8,23 @@ go get github.com/regen-network/cosmos-proto/protoc-gen-gocosmos 2>/dev/null
 # get cosmos sdk from github
 # go get github.com/cosmos/cosmos-sdk@v0.45.11 2>/dev/null
 
+generate_protos() {
```

Comment:
Was regular proto generation not working?

Reply:
I just moved the proto script to it's own function. It was called on just the stride directory previously, but now I had to add the cosmos directory. So rather than doing
```bash
  proto_dirs=$(find ./stride -path -prune -o -name '*.proto' -print0 | xargs -0 -n1 dirname | sort | uniq)
  for dir in $proto_dirs; do
    for file in $(find "${dir}" -maxdepth 1 -name '*.proto'); do
      if grep go_package "$file" &>/dev/null; then
        buf generate --template buf.gen.gogo.yaml "$file"
      fi
    done
  done

  proto_dirs=$(find ./cosmos -path -prune -o -name '*.proto' -print0 | xargs -0 -n1 dirname | sort | uniq)
  for dir in $proto_dirs; do
    for file in $(find "${dir}" -maxdepth 1 -name '*.proto'); do
      if grep go_package "$file" &>/dev/null; then
        buf generate --template buf.gen.gogo.yaml "$file"
      fi
    done
  done
```

I just did 
```bash
generate_protos() {
  package="$1"
  proto_dirs=$(find $package -path -prune -o -name '*.proto' -print0 | xargs -0 -n1 dirname | sort | uniq)
  for dir in $proto_dirs; do
    for file in $(find "${dir}" -maxdepth 1 -name '*.proto'); do
      if grep go_package "$file" &>/dev/null; then
        buf generate --template buf.gen.gogo.yaml "$file"
      fi
    done
  done
}

generate_protos "./stride"
generate_protos "./cosmos"
```

Reply:
this was because I had to add the Detokenize MsgType from LSM

Reply:
Cool that makes sense